index.js

```diff
@@ -531,23 +531,6 @@ RingPop.prototype.onMemberFaulty = function onMemberFaulty(member) {
     this.suspicion.stop(member);
 };
 
-RingPop.prototype.onMemberJoined = function onMemberJoined(member) {
-    this.stat('increment', 'membership-update.new');
-    this.logger.debug('member has joined', {
-        local: this.membership.localMember.address,
-        joined: member.address
-    });
-
-    this.ring.addServer(member.address);
```

Comment:
when a "new" member is added to the membership list, we need to use the status of that member determine how to modify the ring/membership list. Prior to this change, `new` and `alive` were treated the exact same.